index.md

```diff
@@ -16,49 +16,33 @@ order: 1
-
-- Guidelines
-- Specific uses of the tool
-- What authoring tools integrate with the evaluation tool
-- API
-- Browser plugin
-- etc.
-
-This document also discusses some specific usage of evaluation tools, such as:
-- Reports
-- Step-by-step evaluation
-- In-page feedback
-- Page transformation
-
-Finally, this document offers further aspects to consider, such as:
-- Organizational structure and development process
-- Complexity and size of Web site
-- Skills and knowledge of Web developers
-
-*Disclaimer*: WAI encourages the development and evolution of Web accessibility evaluation tools. WAI maintains an extensive [list of evaluation, repair and transformation tools](https://www.w3.org/WAI/ER/tools/). WAI does not endorse or promote any single tool or vendor.
+Web accessibility evaluation tools are software programs or online servicesthat help determine if a Web site is accessible. This document highlights features of evaluation tools, and discusses different aspects to consider when selecting these tools. Links to different tools is provide in the [Web Accessibility Tool List](https://www.w3.org/WAI/ER/tools/).
 
 ## What to Expect from Evaluations Tools
 {:#Expect}
 
-Web accessibility evaluation tools can be a powerful part of a developer's arsenal. You can use evaluation tools through all phases of Web site development. They assist with determining the level of accessibility of Web sites. Evaluation tools can verify the conformance of Web sites for items that can be reviewed automatically. They can also help with manual reviews.
+Web accessibility evaluation tools can help you quickly identify potential accessibility issues. You can use them through all phases of the web design and development process. Tools can provide fully-automated checks, as well as help you with manual review.
 
-But we cannot check all aspects of a site's accessibility automatically. Human judgement is often required. Sometimes evaluation tools can produce false or misleading results. Web accessibility evaluation tools can not *determine* the accessibility of Web sites. They can only *assist* in doing so.
+But we cannot check all accessibility aspects automatically. Human judgement is often required. Sometimes evaluation tools can produce false or misleading results. Web accessibility evaluation tools can not *determine* accessibility, they can only *assist* in doing so.
 
 ## Features of Evaluation Tools
 {:#Features}
 
-Web accessibility evaluation tools often have many features. The features are sometimes similar from one tool to another. Sometimes they are very different. 
+Web accessibility evaluation tools target different audiences. This includes designers, developers, non-technical content authors, quality assurance testers, and sometimes also end-users. Thus tools tend to offer different features and functionality.
```

Comment:
I thought the initial content did not really say much. This is an optimal place to mention some of the different audiences of evaluation tools too.